genomics/seed_extend.py

```diff
@@ -49,6 +58,8 @@ def extract(c, margin, occ_matrix, read, reference, seed, seed_length, scoring_p
                                                       scoring_points)
         alignment, transcript = traceback(read[seed_length:], reference[start_pos:end_pos], distances,
                                           scoring_points)
-        result.append((start, end, dir, alignment_score, transcript))
+
+        if(best_alignment_score < alignment_score):
+            result = (start, end, dir, alignment_score, transcript)
```

Comment:
result not defined

Reply:
Could potentially break the code if for some reason for does not go into a single loop